sys-auth/polkit/files/override.conf

```diff
@@ -0,0 +1,78 @@
+[Service]
```

Comment:
Installing to `/etc` feels wrong for a system package (it might be okay for Flatcar's case because they're shipping something where the package manager isn't then used in the final product).

Does systemd let us install override files to /lib/systemd, or do we need to just patch the Polkit unit file instead?

Reply:
I think it would make more sense to backport the patches from upstream than to introduce a distro-specific override file.

Reply:
I'm personally not on board with applying hardening settings beyond what upstream is willing to support.